dbt_subprojects/daily_spellbook/models/addresses/chains/_schema.yml

```diff
@@ -0,0 +1,68 @@
+version: 2
+
+models:
+  - name: addresses_ethereum_info
+    meta:
+      blockchain: ethereum
+      sector: addresses
+      project: addresses
+      contributors: hildobby
+    config:
+      tags: ['table', 'address', 'info']
+    description: "High level information about every address on Ethereum"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - address
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain"
+      - &address
+        name: address
+        description: "Address"
+      - &executed_tx_count
+        name: executed_tx_count
+        description: "The number of transactions executed by the address"
+      - &max_nonce
+        name: max_nonce
+        description: "The highest nonce value"
+      - &is_smart_contract
+        name: is_smart_contract
+        description: "Is the address a smart contract"
+      - &namespace
+        name: namespace
+        description: "If the address is a smart contract that is decoded on Dune, here is the associated namespace on Dune"
+      - &name
+        name: name
+        description: "If the address is a smart contract that is decoded on Dune, here is the associated smart contract name on Dune"
+      - &first_funded_by
+        name: first_funded_by
+        description: "Which other address first ever funded this address"
+      - &first_tx_block_time
+        name: first_tx_block_time
+        description: "The block time of the first transaction executed by this address"
+      - &last_tx_block_time
```

Comment:
Not sure if this is equivalent to `last_seen`, but this column seems not appeared in the final selection.

Reply:
no this is not the same column, last_tx_block_time is for the last executed tx. Then there's last_seen which is for transfers (executed or not by the address). The two are purposely separate because it's nice to be able to filter for just executed txs by time if thats what you're after